lib/aws-record/record/batch_writer.rb

```diff
@@ -0,0 +1,208 @@
+# Copyright 2015-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not
+# use this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+
+module Aws
+  module Record
+    class BatchWriter < Batch
+      include Enumerable
+
+      # Writer states
+      PENDING_STATE = 'PENDING'.freeze
+      SUCCESSFUL_SEND_STATE = 'SUCCESSFUL'.freeze
+      ERROR_SEND_STATE = 'ERROR'.freeze
+
+      # DynamoDb item request limit
+      ITEM_REQUEST_LIMIT = 25
```

Comment:
No, and i don't see why we would. This is a hard limit relative to the AWS DynamoDb limit of number of requests per batch_write. If that ever changes, then this will need to change.